backend/seed_data/hints.yaml

```
@@ -32,3 +32,27 @@ hints:
         order: 1
       - text: Consider using three pointers (prev, current, and next) to systematically reverse the list's links.
         order: 2
+
+# Binary Trees
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)